rosdep/base.yaml

```
@@ -495,6 +495,11 @@ catch2:
   ubuntu:
     '*': [catch2]
     focal: null
+cbindgen:
+  ubuntu:
+    focal: [cbindgen]
+    jammy: [cbindgen]
```

It's not actually in Ubuntu Focal, but it's in Jammy and later, so this should be:

```suggestion
  ubuntu:
    '*': [cbindgen]
    focal: null
```

https://packages.ubuntu.com/noble/cbindgen